src/aliceVision/sfmData/colorize.cpp

```diff
@@ -12,17 +12,51 @@
 #include <aliceVision/stl/mapUtils.hpp>
 #include <aliceVision/image/io.hpp>
 #include <aliceVision/system/ProgressDisplay.hpp>
+#include <aliceVision/system/Timer.hpp>
 
 #include <map>
 #include <random>
 #include <vector>
 #include <functional>
 namespace aliceVision {
+
+class LMColorAccumulator
+{
+    public:
+        LMColorAccumulator()
+            : rgbFinal(0.0)
+            , sumSegments(0.0)
+            {}
+        ~LMColorAccumulator() {}
+
+
+        image::RGBfColor rgbFinal;
+        double sumSegments;
```

Comment:
prefer member initialization and default constructor
```suggestion
        image::RGBfColor rgbFinal{0};
        double sumSegments{.0};
```